content/articles/how-to-make-a-discord-bot-using-python/index.md

```diff
@@ -0,0 +1,292 @@
+
+Bot users are automated programs that appear and behave like people and respond to events and orders on Discord automatically. Users of Discord bots (or just bots) have virtually limitless options.
+
+Let's assume you're in charge of a new Discord guild and a new member joins for the first time. If you're feeling very enthusiastic, you may personally contact that individual and welcome them to your community. You may also inform them about your channels or request that they introduce themselves.
+
+When an individual feels welcomed and likes the conversations in your guild, therefore they invite their friends.
+
+The guild's membership expands to the point that it's no longer practical to reach each new member individually. Still, you always want to send them something that will acknowledge their arrival.
+
+It's possible to automatically respond to a new member entering your guild using a bot. You may even manage how it interacts with each new user by customizing its behavior based on circumstances.
+
+This is fantastic, but it's only an example of how a bot may be beneficial. Once you learn how to build bots, there are several ways to get creative with them.
+
+**Note: Even though Discord permits you to develop bots for vocal communication, this tutorial will focus on the texting side of the platform.**
+
+When building a bot, there are two critical stages to follow:
+
+1. Create a Discord bot user and assign it to a guild.
+2. Create code that executes your bot's actions and makes use of Discord's APIs.
+
+In the following part, you'll discover how to use Discord's Developer Portal to create a Discord bot.
+
+###  Overview and requirements
+
+We'll be performing all of our codings and hosting our bot using the Repl.it web IDE, so there won't be any other software to set up in your system. However, you'll need to establish a Discord account for this guide (You may skip this if you already have one). The following section contains instructions on how to achieve this.
+
+We'll go through the following points in this tutorial:
+* Build an application and a bot user in your Discord account.
+* Creating a server on Discord. 
+* Joining the Discord server with our bot.
+  
+Let's start with the administrative tasks, and then we'll move on to the exciting part: *programming our bot*.
+
+### Creating a discord bot account on the Discord
+
+We need to create a Discord Bot account before using the Python library and the Discord API.
+
+To begin, you'll need to build a few Discord components:
+
+1.	An account
+2.	An application
+3.	A bot
+4.	A guild
+
+In the sections that precede, you'll discover much more about these components.
+
+When you've completed each of these components, you'll need to register your bot with your guild to connect everything.
+
+You may begin by entering [Discord's User Portal](https://discord.com/login?redirect_to=%2Fdevelopers%2Fapplications)
+
+1.	**Creating an Account** - 
```

Comment:
Change the formatting to the H4 header. Remove the bold operator as well.

#### Creating an account

Let the first letter be capitalized, and the rest all start with small letters.